metabrainz/templates/index/gdpr.html

```diff
@@ -154,8 +154,8 @@ <h2>{{ _('Right to data portability / exporting your data') }}</h2>
      <li>
         <b>IP addresses</b>: If you visit one of our sites the IP address of your computer is stored in our web logs for 7 days 
         before our servers automatically delete them. This data is almost never used by anyone: the only time we look at the 
-        IP addresses is if our sites are adversely impacted by the actions of some unknown users. In that case we may investigate 
-        the logs in order to identify the IP address that needs to be blocked in order to restore the stability of our service. 
+        IP addresses is if our sites are adversely impacted by the actions of some unknown users. In that case, we may investigate 
+        the logs to identify the IP address that needs to be blocked in order to restore the stability of our service. 
```

Comment:
Original sounds like it might be some legalese? Doesn’t seem like any information is being lost by removing this bit though, so if there are no other objections to removing it, 👍 from me.